opendcs-rest-api/src/test/java/org/opendcs/odcsapi/res/AppResourcesTest.java

```diff
@@ -0,0 +1,117 @@
+package org.opendcs.odcsapi.res;
+
+import java.time.Instant;
+import java.util.Date;
+import java.util.Properties;
+
+import decodes.sql.DbKey;
+import decodes.tsdb.CompAppInfo;
+import decodes.tsdb.TsdbCompLock;
+import org.junit.jupiter.api.Test;
+import org.opendcs.odcsapi.beans.ApiAppRef;
+import org.opendcs.odcsapi.beans.ApiAppStatus;
+import org.opendcs.odcsapi.beans.ApiLoadingApp;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.opendcs.odcsapi.res.AppResources.map;
+import static org.opendcs.odcsapi.res.AppResources.mapLoading;
+
+final class AppResourcesTest
+{
+	@Test
+	void testAppRefMap()
+	{
+		CompAppInfo compAppInfo = new CompAppInfo();
+		compAppInfo.setAppName("Computation application");
+		compAppInfo.setComment("Computation to find the volume of a river");
+		compAppInfo.setLastModified(Date.from(Instant.parse("2021-07-01T00:00:00Z")));
+		compAppInfo.setAppId(DbKey.createDbKey(151615L));
+		compAppInfo.setNumComputations(1);
+		Properties properties = new Properties();
+		properties.setProperty("compRef", "applicationValue");
+		compAppInfo.setProperties(properties);
+
+		ApiAppRef appRef = map(compAppInfo);
+
+		assertNotNull(appRef);
+		assertEquals(compAppInfo.getAppName(), appRef.getAppName());
+		assertEquals(compAppInfo.getComment(), appRef.getComment());
+		assertEquals(compAppInfo.getLastModified(), appRef.getLastModified());
+		assertEquals(compAppInfo.getAppId().getValue(), appRef.getAppId());
+		assertEquals(compAppInfo.getAppType(), appRef.getAppType());
```

Comment:
Somewhere in this method should be a comment about Properties.  ApiAppRef doesn't have a place to hold the properties -right?   Its good to show that map() isn't broken by presence/absence of Properties on the input but I think it would be nice to have a comment pointing out that they aren't mapped.

Reply:
Added a properties comment and notes for other data lost in mapping